pages/index.tsx

```diff
@@ -1,116 +1,8 @@
-import { Box, Button, Grid, Typography } from '@mui/material';
-import {
-  School as SchoolIcon,
-  Lightbulb as LightbulbIcon,
-} from '@mui/icons-material';
 import Head from 'next/head';
```

Comment:
I've been keeping the imports from npm at the top. Then using a blank line. And then imports from our codebases.  Lets keep it this way for consistency. 
```suggestion
import Head from 'next/head';

```